script.js

```diff
@@ -9,3 +9,26 @@
 // Add a `facial-hair` property to `instructor`.
 
 // Log the value of the `facial-hair` property to the console.
+
+
+let colors= ["red", "yellow", "green"]
+
+
+let green = colors[2]
+
+for (index = 0; index < colors.length; ++index){
+  console.log("my favorite color is")
+}
+// colors.forEach(fucntion(find)){
+  //console.log("my favorite color is .find")
+    //not so sure if this is correct
+
+let instructor = {
+  gender: male;
+  firstname: john;
+  hair: long;
+}
+
+instructor.facialhair = "got some"
+
+console.log(facialhair)
```

Comment:
going to need to access the `facialhair` property on the instructor object so `console.log(instructor.facialhair)`